parastell/nwl_utils.py

```diff
@@ -187,7 +187,9 @@ def flux_coords(plas_eq, wall_s, coords, num_threads):
             for theta_coord in theta_coord_chunk
         ]
     # Ensures theta_coords are all positive (add 360 degrees where needed)
-    theta_coords = (theta_coords + 2 * np.pi) % 2 * np.pi
+    theta_coords = [
+        (theta_coord + 2 * np.pi) % 2 * np.pi for theta_coord in theta_coords
+    ]
```

Comment:
Might be better to change `theta_coords` to a NumPy array

Reply:
Moving to arrays rather than lists would be somewhat large rework, as we make use of append to deal with the variable length iterables that come from the multithreading approach, I'd prefer to keep this PR as a minimal change to get things into a working state.

Regardless, I realized we can just put this coordinate shift in the function where we calculate theta, which I think is a bit more elegant.

Reply:
The whole nwl_utils could possibly be reworked similar to what Connor has done for radial_distance_utils, but I think that would be better for a different PR.